app/models/generic_event.rb

```diff
@@ -0,0 +1,18 @@
+class GenericEvent < ApplicationRecord
+  CREATED_BY_OPTIONS = %w[serco geoamey unknown].freeze
+
+  belongs_to :eventable, polymorphic: true, touch: true
```

Comment:
do we not need the reverse `has_many` on moves?

Reply:
Not currently needed. Probably will be needed in a later PR around running through triggers on each event. Laterbase